### ISSUE TITLE
Peer Review by Group 98 - Divya, Gauravpreet and Aamishi

### ISSUE THREAD

**Issue by aamishi** [wyx827/2024USpresidentialelection#4]
## Summary
I think that you are off to a wonderful start. The appendix and model section seems to have major work done. You only have the writing part left as most of your statistical work is complete.
## Strong positive points:
•	Appendix is phenomenal, good job. I think that you chose the correct things to focus on for the draft submission.
•	Model section looks good. As most of the paper is underway, make sure to follow the rubric to ensure that you are meeting all the requirements of the paper.
## Critical improvements needed:
•	Most of the paper is still incomplete. If you plan on not submitting this paper, be sure to at least cite R and other essentially citations.
•	Please update the R and Data Citation, LLM section and README.md. These are no-go’s so even if you have a wonderful paper, it would not be checked unfortunately.
•	The pdf needs a lot of improvement. Make sure that there is no raw R code on the paper.
•	Use a global seed instead of a local seed with the same value.
## Suggestions for improvement:
•	Anytime you use function or variable names like trump_win, format it to look like code. The model section is visually unappealing and cluttered making it hard to follow.
•	You can remove file extensions like .log and .aux, etc. from your paper folder. These files are usually of no significance to a statistics research paper.
•	There are minor inconsistencies in the headings sizing.

## Evaluation:
R is appropriately cited (1 pts):
0/1: No citations yet

Data are appropriately cited (1 pts):
0/1: 0, minimal writing is complete but please ensure that you cite the data that you use.

Class paper (1 pts):
0/1: Mostly no evidence of being a class paper, except in the README file which is directly from the starter code

LLM usage documented (1 pts):
0/1: No mention of LLM usage or non-usage. Please update your files and README. Mention the specific LLM used.

Title (2 pts):
0/2: No title added. Include information title and subtitle.

Author, date, and repo (2 pts):
1/2: Mention of “First Author”. Please remove this if only 2 members are in the group. Git Repo looks good.

Abstract (4 pts):
0/4: No abstract.

Introduction (4 pts):
0/4: No introduction included yet.

Estimand (1 pts):
0/1: Estimand not mentioned in the paper

Data (10 pts):
4/10: I think that the group has started good work with the data. The simulations, download, cleaning and testing are all in place. Explicitly mention the source of the data. Good work with comments.

Measurement (4 pts):
0/4: No measurement yet

Model (10 pts):
3/10: Model is set up well for logistic regression. Each variable is also included. I would suggest to use a table instead of overflowing text while explaining the predictor variables. Include the software that was used to generate the model.

Results (10 pts):
0/10: No results included

Discussion (10 pts):
0/10: No discussion included

Prose (6 pts):
2/6: As writing is very limited, I cannot judge the quality of the prose very well. The model section seems to be very concise and free of flowery language. Keep it up for the rest of the paper.

Cross-references (1 pts):
0/1: Cross-references are not included

Captions (2 pts):
0/2: No captions/figures included

Graphs/tables/etc (4 pts):
0/4: No graphs or tables included

Idealized methodology (10 pts):
0/10: No methodology included yet

Idealized survey (4 pts):
0/4: Not included

Pollster methodology overview and evaluation (10 pts):
9/10: Well-written methodology.

Referencing (4 pts):
0/4: No references yet. CITE R!!!

Commits (2 pts):
2/2: More than 2 commits with appropriate messages provided

Sketches (2 pts):
0/2: The paper does not have sketches.

Simulation (4 pts):
4/4: Simulations are well done. Include a global seed.

Tests - simulation (4 pts):
4/4: Appropriately simulated and tested the data

Tests - actual (4 pts):
4/4: Well-done! More than 10 tests included! Cite LLM usage if used!

Parquet (1 pts):
1/1: Analysis data is saved as a parquet which is used for the model.

Reproducible workflow (4 pts):
4/4: All scripts are well-written and look reproducible.

Miscellaneous (3 pts):
0/3: Unfinished

## Estimated overall mark:
38/126 but 0/126 because no R/LLM citation is included.

## Any other comments:
You’re set up for success so please make sure silly reasons like R and LLM citations do not withhold your grade! All the best.

**Comment by wyx827** [comment]
We have fixed the issues you mentioned Thanks!
